ocs_ci/utility/pagerduty.py

```diff
@@ -45,6 +47,41 @@ def set_pagerduty_integration_secret(integration_key):
     logger.info("New integration key was set.")
 
 
+def set_pagerduty_faas_secret(integration_key):
+    """
+    Update managed-fusion-agent-config secret. This is valid only on Fusion aaS platform.
+    managed-fusion-agent-config secret is expected to be present prior to the update.
+
+    Args:
+        integration_key (str): Integration key taken from PagerDuty Prometheus integration
+
+    """
+    logger.info("Setting up PagerDuty")
+    kubeconfig = os.getenv("KUBECONFIG")
```

Comment:
I am not sure, but does this work in a multicluster configuration ?
Alternative:
```
kubeconfig = os.path.join(
            config.ENV_DATA["cluster_path"], config.RUN.get("kubeconfig_location")
        )
```

Reply:
I am not sure but those tests are currently not working correctly for multicluster. I am planning to update those tests for multicluster and to be run with other tests from consumer which is not implemented yet in a future PR.